stream_alert/rule_processor/main.py

```diff
@@ -18,9 +18,6 @@
 import os
 
 from stream_alert.rule_processor.handler import StreamAlert
-from rules import (
```

Comment:
if we are getting rid of this, we need an action item to standardize loading matchers from within the loop below